src/ja/server/proxy/proxy.py

```diff
@@ -0,0 +1,38 @@
+from ja.common.proxy.proxy import ContinuousProxy
+from ja.common.proxy.ssh import SSHConfig
+from ja.common.message.worker import WorkerResponse
+
+
+class WorkerProxy(ContinuousProxy):
+    """
+    Proxy for the worker client used on the central server.
+    """
+    def __init__(self, ssh_config: SSHConfig):
+        """!
+        @param ssh_config: Config for paramiko.
+        """
+
+    # TODO replace object with Job class
```

Comment:
Job was merged.

Reply:
This PR also depends on my class stubs UserClientCLI.
I'll rebase and fix this once that PR is merged.